cyences_app_for_splunk/default/macros.conf

```diff
@@ -1365,6 +1365,24 @@ definition = search *
 [cs_aws]
 definition = index IN (aws,summary)
```

Comment:
Is `cs_aws_cloudtrail_logs` macro present already?

Reply:
Yes, it is present.